android/fakewallet/src/main/java/com/solana/mobilewalletadapter/fakewallet/MobileWalletAdapterActivity.kt

```diff
@@ -31,6 +32,23 @@ class MobileWalletAdapterActivity : AppCompatActivity() {
             }
         }
 
+        lifecycleScope.launch {
+            repeatOnLifecycle(Lifecycle.State.CREATED) {
+                viewModel.lowPower.collect {
+                    // should use dialog fragment, etc. but this is a quick demo
+                    AlertDialog.Builder(this@MobileWalletAdapterActivity)
+                        .setTitle(R.string.low_power_mode_warning_title)
+                        .setMessage(R.string.str_low_power_mode_warning_dsc)
+                        .setPositiveButton(R.string.label_return_to_dapp) { _, _ ->
```

Comment:
MAYBE: use the standard [android.R.string.ok](https://developer.android.com/reference/android/R.string#ok) here